### ISSUE TITLE
PR #1365 broke importing diffusers

### ISSUE THREAD

**Issue by OrionFive** [huggingface/diffusers#1375]
### Describe the bug

When I try to import diffusers, I get the error below. The file causing the error is from PR #1365, which was merged today.

### Reproduction

```
%cd /content
!git clone -b main  https://github.com/huggingface/diffusers.git
import sys
sys.path.insert(0,'/content/diffusers/src/')
import diffusers
```

### Logs

```shell
ImportError                               Traceback (most recent call last)
<ipython-input-2-3ef39c283978> in <module>
     20 import sys
     21 sys.path.insert(0,'/content/diffusers/src/')
---> 22 import diffusers
     23 diffusers.__file__
     24 

5 frames
/content/diffusers/src/diffusers/pipelines/stable_diffusion/pipeline_stable_diffusion_image_variation.py in <module>
     20 import PIL
     21 from diffusers.utils import is_accelerate_available
---> 22 from transformers import CLIPFeatureExtractor, CLIPVisionModelWithProjection
     23 
     24 from ...models import AutoencoderKL, UNet2DConditionModel

ImportError: cannot import name 'CLIPVisionModelWithProjection' from 'transformers' (/usr/local/lib/python3.7/dist-packages/transformers/__init__.py)
```

### System Info

On a colab notebook.

**Comment by ki-arie** [comment]
Stopgap: `pip install diffusers==0.7.2`

**Comment by OrionFive** [comment]
I suppose pipeline_stable_diffusion_image_variation.py  should fail gracefully if an unreleased model it is depending on is not available.

**Comment by anton-l** [comment]
Fixed in a patch release `0.8.1`: https://github.com/huggingface/diffusers/releases/tag/v0.8.1
:warning: Please upgrade with `pip install --upgrade diffusers` or `pip install diffusers==0.8.1`

**Comment by patrickvonplaten** [comment]
@OrionFive, you are 100% right - we really messed up this release. This should have not happened! Very sorry about this. 

It's the first time we are depending on a unreleased version of a soft dependency (`"transformers"`) and have for some reason completely forget that most people are not working with the main version of "transformers", but the latest "pip version". Also because it's a soft dependency our tests didn't catch it. 

Everything should work as expected. Thanks a mille for reacting so quickly here @anton-l and @pcuenca 

Super sorry about this :pray: - we should really know better here! 

We will make sure to test this better going forward and not have such a broken release again.
